src/main/java/com/softserve/demo/model/Customer.java

```diff
@@ -47,4 +51,13 @@
     @JsonBackReference(value = "order_customer")
     @OneToMany(fetch = FetchType.LAZY, mappedBy = "customer",cascade = CascadeType.REMOVE)
     private List<Order> orders;
+
+    @ManyToMany(cascade = {
```

Comment:
I think this mapping is wrong. First of ALL add fetch type lazy, second cascadeType.ALL is bad idea for many to many mappings. Please check other mapping in project, guess we need only Remove cascade here